jobManager.py

```diff
@@ -82,6 +82,9 @@ def __manage(self):
         elif Config.VMMS_NAME == "ec2SSH":
             from vmms.ec2SSH import Ec2SSH
             vmms = Ec2SSH()
+        elif Config.VMMS_NAME == "dockerSSH":
```

Comment:
Regretfully, this naming scheme no longer makes much sense, as we're not using SSH anymore.